### PR TITLE
Add solairerove user

### PR DESCRIPTION
**Pull request by solairerove**

### CODE REVIEW

File: conf/users/solairerove.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module "solairerove" {
+  source  = "git::https://github.com/cloudposse/terraform-aws-iam-user.git?ref=tags/0.1.1"
+  name    = "solairerove"
```
Comment (osterman) [inline conf/users/solairerove.tf]: Actually, let's use your cloudposse email address. I missed this in my previous reviews.

File: conf/users/solairerove.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  pgp_key = "keybase:solairerove"
```
Comment (osterman) [inline conf/users/solairerove.tf]: Have you created this keybase user?

File: conf/users/solairerove.tf (hunk 1 above)
Comment (solairerove) [reply conf/users/solairerove.tf]: name    = "mikita@cloudposse.com"

**Comment by osterman** [comment]
atlantis/root plan

**Comment by osterman** [comment]
atlantis/root apply -p users

**Pull request merged**